rules-tattletale/addon/src/main/java/org/jboss/windup/rules/apps/tattletale/TattletaleRuleProvider.java

```diff
@@ -145,5 +147,18 @@ private void createReportModel(GraphContext context, FileModel input, String rep
             ReportService reportService = new ReportService(context);
             reportService.setUniqueFilename(applicationReportModel, "tattletale" + "_" + inputProjectModel.getName(), "html");
         }
+
+        private boolean isTattleTaleReportGenerationEnabled(GraphRewrite event)
+        {
+            Boolean enableReport = (Boolean) event.getGraphContext().getOptionMap().get(EnableTattletaleReportOption.NAME);
```

Comment:
We could convert this to :
`
            boolean enableReport = (boolean) event.getGraphContext().getOptionMap().getOrDefault(EnableTattletaleReportOption.NAME, Boolean.FALSE);`

Reply:
I'm afraid this would lead to a NPE in case there's no such a key due to the unboxing done on `null`

Reply:
Ah sorry, you proposed the default version even if i don't think we should have the options default here.
Would you prefer having the `boolean` instead of `BooleanUtils` for readability?

Reply:
well, a pure boolean is easier to read, and in fact if we dont have the value in the map we consider them as false, is what BooleanUtils is doing 
`public static boolean isTrue(Boolean bool) {
        if (bool == null) {
            return false;
        } else {
            return bool;
        }
    }`